debian/changelog

```diff
@@ -0,0 +1,12 @@
+eos-data-distribution (0.1.0) UNRELEASED; urgency=medium
```

Comment:
(If you still want this done, I’ll split out the commit before merging, to avoid having too many pending branches.)